docs/modules/ROOT/pages/references/architecture/control-api-billing-entity.adoc

```diff
@@ -0,0 +1,99 @@
+= {controlapi}: BillingEntity
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_manage_billing_information["Manage Billing Information"] features.
+
+It is required that billing data can be modified by the billing entity owner in the portal and by our accounting team in the ERP.
+
+A naive implementation with a controller that synchronizes the `BillingEntity` resource with the ERP would be problematic.
+An implementation based on a controller would create a system with multiple sources of truth.
+In the general case, there's no right order for independent updates in such a system.
+
+To ensure we have a single source of truth the `BillingEntity` is a virtual resource that is stored in the ERP and fetched by an aggregated API server.
+
+The cluster's etcd, a ConfigMap, or a CRD may be used as a cache for information fetched from the ERP.
+
+Having a virtual resource also helps with filtering of list operations on Kubernetes resources.
+
+List operations on Kubernetes resources can't be filtered by RBAC, it's a binary operation:
+Resources can either be listed or not, there is no way to give access only to a sub-set.
```

Comment:
Those feel like random thoughts and I fail to see context and purpose why they are here.